src/test/resources/application-test.yml

```diff
@@ -1,5 +1,8 @@
 spring:
 
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
This is required for the the test configuration class TestEmailConfiguration.

Reply:
But what does it all mean, Matt? Are we overriding tests with a different bean definition?

Reply:
No just in the one integration test for email testing.